string command
---
int8 Result